ExomeDepth/make_BEDdetail.py

```diff
@@ -0,0 +1,365 @@
+                            excluded_samples_file.write("Sample {sampleid} run {runid} is excluded because not all QC are above threshold\n".format(
+                                sampleid=sampleid,
+                                runid=runid
+                                ))
+                            break
+
+                        """ Event stats """
+                        chrom = splitline[0]
+                        start = splitline[1]
+                        stop = infofields[2].split("=")[1] 
+                        calltype = infofields[0].split("=")[1]
+                        ntargets = infofields[1].split("=")[1]
+                        svlen = infofields[3].split("=")[1]
+                        ratio = formatfields[3]
+                        ccn = formatfields[1]
+                        bf = formatfields[2]
+                        all_event_file.write("{sampleid}\t{chrom}\t{start}\t{stop}\t{gender}\t{refset}\t{calltype}\t{ntargets}\t{svlen}\t{ratio}\t{ccn}\t{bf}\t{correl}\t{deldupratio}\t{totalcalls}\t{refsamples}\n".format(
+                            sampleid=sampleid, 
+                            chrom=chrom,
+                            start=start,
+                            stop=stop,
+                            gender=gender,
+                            refset=refset,
+                            calltype=calltype,
+                            ntargets=ntargets,
+                            svlen=svlen,
+                            ratio=ratio,
+                            ccn=ccn,
+                            bf=bf,
+                            correl=correl,
+                            deldupratio=deldupratio,
+                            totalcalls=totalcalls,
+                            refsamples=refsamples
+                        ))
+                        event = "{chrom}_{start}_{stop}_{calltype}_{ntargets}".format(
+                            chrom=chrom,
+                            start=start,
+                            stop=stop,
+                            calltype=calltype,
+                            ntargets=ntargets
+                            )
+                        if event not in event_dic:
+                            event_dic[event] = {"parent":{"count":0, "bf":[],"ratio":[],"correlation":[],"deldupratio":[],"totalcalls":[], "gender":[]}, \
+                                            "child":{"count":0, "bf":[],"ratio":[],"correlation":[],"deldupratio":[],"totalcalls":[], "gender":[]}}
+
+                        """ Determine Child or Parent status based on sampleid. There is no other option at the moment"""
+                        if "CM" in sampleid or "CF" in sampleid or "CO" in sampleid:
+                            sampletype = "child"
+                            childs +=1
+                        elif "PM" in sampleid or "PF" in sampleid or "PO" in sampleid:
+                            sampletype = "parent"
+                            parents += 1
```

Comment:
Checken of er 'P' of 'C' in sampleid staat is voldoende?

Reply:
in principe wel, maar er is een grotere kans dat er toevallig een C of P in de samplenaam staat dan CM/CP etc.
Bijvoorbeeld bij Controle sample zou het mis kunnen gaan, al zullen die niet in de dataset moeten zitten door de prefiltering. Maar zoals het nu is lijkt mij veiliger.